db/migrate/20200311201103_add_moderator_id_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddModeratorIdToComments < ActiveRecord::Migration[5.2]
```

Comment:
should this be here?

Reply:
Hmm yeah I guess this could be moved into a different PR 👍